README.md

```diff
@@ -36,11 +47,68 @@ This project uses NPM for dependency management and Forge for tests so you'll ne
 
     > **_NOTE:_**  You may need to specify manually maximum allowed requests per second (rps) value for an API key/RPC url for some providers. In our experience max 100 rps will be enough to run tests.
 
-## Running tests
 
-```sh
-forge test
-```
+## Running Tests
+
+This repository contains different sets of tests written using the Foundry framework:
+
+- **Unit tests** - Basic tests that cover each module in isolation. This is the most comprehensive set of tests, covering every edge case and aspect of the code.
+
+- **Integration tests** - Tests that verify how contracts work in a forked environment using the real state of the protocol. These tests are split into two subcategories:
+    - **Scenario tests** - Usually edge cases that demonstrate how the system behaves under very specific conditions. These tests use fresh deployments of the DG contracts (and forked Lido contract state) to prepare the system state for concrete scenarios.
+    - **Regression tests** - Integration tests that can be launched on forked instances of the DG contracts. These tests verify the required functionality of the system and ensure everything works exactly as expected. This category contains two special tests: 
+        1) A [test for complete rage quit](./test/regressions/complete-rage-quit.t.sol) of the majority of real stETH and wstETH holders. 
+        2) A special [solvency test](./test/regressions/dg-solvency-simulation.t.sol) that simulates protocol operation under conditions with serial Rage Quits. 
+    
+            Both tests may require significant time to complete, so they are not expected to be run frequently, in contrast to regular regression tests which can be run daily to verify the system works correctly.
+
+The following commands can be used to run different types of tests:
+
+- **Run unit tests exclusively**
```

Comment:
List of required env variables is necessary to be listed here

Reply:
Fully agree! Propose adding this via a standalone PR to avoid increasing the scope of the current one.